test/hotspot/jtreg/runtime/ErrorHandling/CreateCoredumpOnCrash.java

```diff
@@ -52,15 +52,17 @@ public static void main(String[] args) throws Exception {
             runTest("-XX:-CreateMinidumpOnCrash").shouldContain("CreateCoredumpOnCrash turned off, no core file dumped")
                                                  .shouldNotHaveExitValue(0);
         } else {
-            runTest("-XX:+CreateCoredumpOnCrash").shouldNotContain("CreateCoredumpOnCrash turned off, no core file dumped")
-                                                 .shouldNotHaveExitValue(0);
+          OutputAnalyzer oa = runTest("-XX:+CreateCoredumpOnCrash");
+          oa.reportDiagnosticSummary();
+          oa.shouldContain("core dump info").shouldNotContain("CreateCoredumpOnCrash turned off, no core file dumped").
+                  shouldNotHaveExitValue(0);
```

Comment:
Nit: you should call `reportDiagnosticSummary` last otherwise on failure it will be called twice: once directly  by the test and then indirectly by the failing "should" function.